docs/pages/architecture/tls-routing.mdx

```diff
@@ -157,6 +148,34 @@ Teleport proxy's web listener and don't require a local proxy or any special
 ALPN/SNI negotiation. These web connections use standard `http1.1` and `h2`
 protocols for ALPN.
 
+Application access for TCP applications connects through a local proxy by `tsh
+proxy app` where the local proxy forwards the traffic to Teleport Proxy service
+using the `teleport-tcp` ALPN protocol.
```

Comment:
Same here. I think that this the tsh proxy app is mostly useful in case of L7 LB flow but in default TLS Routing flow the `tsh
proxy app` is not needed.

Reply:
For TCP app, i believe we only suggest using a local proxy:
https://goteleport.com/docs/application-access/guides/tcp/#step-44-connect